standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+
+This document should therefore be read in conjunction with the Mobile Application Guidance.
+
+Which Mobile Platforms Must Be Supported
+----------------------------------------
+
+Mobile applications must run on mobile devices. Currently DEFRA has adopted Apple technology as the standard platform for mobile devices. **All mobile applications intended for internal use must therefore run on Apple devices. All mobile applications intended for use beyond the DEFRA family, by any form of "extended enterprise" or the general public, must run on at least Apple and Android devices.**
+
+Working Offline
+---------------
+
+Users of mobile devices, and especially DEFRA family field workers who will often be working in remote, rural locations, must be able to continue to use any mobile applications even when lack of signal prohibits any form of internet access. In fact, **continuing to be readily usable without an internet connection, including the ability to work with data locally on the device, must be considered a defining characteristic of a truly mobile app.**
+
+Extra Security Considerations
+-----------------------------
+
+Application and data security is always essential, and mobile apps can be especially exposed to security risks and challenges. All mobile devices suffer from poor physical security in terms of being easily lost or stolen, and all have to operate over potentially insecure public networks. Furthermore, when an app is intended to run on a public device, then the security of the device is an unknown, and, by default, we should assume the device is inherently insecure. This means that the points below **must be considered and addressed very early in any project to develop such a mobile app**, as they could even influence the practical viability of the app or the scope of the requirements it can securely deliver.
+
+As mobile applications will usually be accessing the internet over entirely public networks **all traffic to and from the app must be encrypted**, for example via HTTPS/Transport Layer Security and/or via the use of a Virtual Private Network (VPN) and/or in-built application security wrapper. Please consult the latest security standards as to the appropriate level of encryption any such mechanisms should adopt, given the confidentiality marking of the data in question.
+
+**If your app, which will normally be operating across the public internet, needs to access DEFRA family systems that are non-public, developers and/or application architects must consult with the Cloud Mobile Services team in Group Infrastructure and Operations**, who will advise on what is or isn't permissible and, where permissible, what mechanisms they can provide to allow such access to be done securely. For example, Cloud Mobile Services may advise as to VPN software and VPN end-points that they can make available to mobile devices that can allow safe access into "back-end", non-public DEFRA family systems in the cloud or on premise.
```

Comment:
How about just "You must consult with Cloud Mobile Services to design the security profile of your app"?

Only for internal ones though?

Reply:
OK, as agreed elsewhere.